app/globals.css

```
@@ -1,5 +1,15 @@
 @import 'tailwindcss';
 
+@layer base {
+  @font-face {
+    font-family: 'ChosunGs';
```

Stylelint (17.11.0)

[error] Expected no quotes around "ChosunGs" (font-family-name-quotes)

(font-family-name-quotes)